code/obj/submachine/robotics.dm

```diff
@@ -122,20 +122,85 @@
 
 /obj/item/atmosporter
 	name = "Atmospherics Transporter"
-	desc = "Used by Atmospherics Cyborgs for convenient transport of siphons and tanks."
-	icon = 'icons/obj/items/items.dmi'
-	icon_state = "bedbin"
+	desc = "Used by cyborgs for convenient transport of siphons and tanks."
+	icon = 'icons/obj/items/device.dmi'
+	icon_state = "atmosporter"
 	var/capacity = 2
 
 	attack_self(var/mob/user as mob)
 		if (src.contents.len == 0) boutput(user, "<span class='alert'>You have nothing stored!</span>")
 		else
 			var/selection = input("What do you want to drop?", "Atmos Transporter", null, null) as null|anything in src.contents
 			if(!selection) return
-			selection:set_loc(user.loc)
-			selection:contained = 0
+			if (istype(selection, /obj/machinery/fluid_canister))
+				var/obj/machinery/fluid_canister/S = selection
+				S.set_loc(user.loc)
+				S.contained = 0
+			else if (istype(selection, /obj/machinery/portable_atmospherics))
+				var/obj/machinery/portable_atmospherics/S = selection
+				S.set_loc(user.loc)
+				S.contained = 0
+			else return //no sparks for unintended items
 			elecflash(user)
 
+
+
+/obj/item/borg_lamp_manufacturer
+	name = "Miniaturized Lamp Manufacturer"
```

Comment:
Appreciate that this is not standardized even remotely, but this should be lowercased,
```suggestion
	name = "miniaturized lamp manufacturer"
```